message_ix/model/MESSAGE/parameter_def.gms

```diff
@@ -450,17 +452,17 @@ Parameters
 
 Parameters
     historical_new_capacity(node,tec,year_all)           historical new capacity
-    historical_activity(node,tec,year_all,mode,time)     historical acitivity
+    historical_activity(node,tec,year_all,mode,time)     historical activity
 ;
 
 ***
 * Auxiliary investment cost parameters and multipliers
 * ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 *
-* Auxilary investment cost parameters include the remaining technical lifetime at the end of model horizon in addition to the
-* different scaling factors and  multipliers as listed below. These factors account for remaining capacity or construction time of new capcaity, 
-* the value of investment at the end of mdoel horizon or the discount factor of remaining lifetime beyond model horizon. 
-* 
+* Auxiliary investment cost parameters include the remaining technical lifetime at the end of model horizon (``beyond_horizon_lifetime``) in addition to the
+* different scaling factors and multipliers as listed below. These factors account for remaining capacity (``remaining_capacity``) or construction time of new capacity (``construction_time_factor``),
+* the value of investment at the end of model horizon (``end_of_horizon_factor``) or the discount factor of remaining lifetime beyond model horizon (``beyond_horizon_factor``).
+*
```

Comment:
please @GamzeUnlu95 have a look if I inserted the right parameters into the right pieces of the paragraph. It was not clear for me that's why I wanted to explicitly relate the text to the parameters listed below.